src/VotesPartialDelegationUpgradeable.sol

```diff
@@ -296,7 +311,7 @@ abstract contract VotesPartialDelegationUpgradeable is
       if (i == 0 && _newDelegations[i]._delegatee == address(0)) {
         // zero delegation is allowed if in 0th position
       } else if (_newDelegations[i]._delegatee <= _lastDelegatee) {
-        revert("VotesPartialDelegation: delegatees must be sorted with no duplicates");
+        revert DuplicateOrUnsortedDelegatees();
```

Comment:
could be helpful to pass the offending address (`newDelegations[i]._delegatee`)

Reply:
true, added in https://github.com/voteagora/ERC20VotesPartialDelegationUpgradeable/pull/40/commits/ad8a10b1ae70d05dc3391b1ffce8c6d5be7894ff